CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changes by Version
 ==================
 
+0.30.0 (2021-07-15)
+-------------------
+* Bumped OpenTelemetry Collector to v0.30.0
+* Add CI check for auto-generated files ([#336](https://github.com/open-telemetry/opentelemetry-operator/pull/336), [@VineethReddy02](https://github.com/VineethReddy02))
```

Comment:
I think this should be left out. The changelog should contain only entries relevant to our end users.